src/MyPractices/Test.java

```diff
@@ -4,6 +4,6 @@ public class Test {
 
     public static void main(String[] args) {
         System.out.println("Hello World");
-        System.out.println("Im a professional Tester");
+        System.out.println("Test Updated");
```

Comment:
which test?

Reply:
hello

Reply:
hello hello, which test updated?

Reply:
Test class updated

Reply:
oh nice thats , I will approve it no need to change